src/wallet/rpcwallet.cpp

```diff
@@ -1571,6 +1571,126 @@ UniValue listtransactions(const JSONRPCRequest& request)
+            "     \"timereceived\" : ttt,    (numeric) The time received in seconds since epoch (1 Jan 1970 GMT)\n"
+            "     \"bip125-replaceable\": \"yes|no|unknown\",  (string) Whether this transaction could be replaced due to BIP125 (replace-by-fee);\n"
+            "                                                      may be unknown for unconfirmed transactions not in the mempool\n"
+            "     \"details\" : [\n"
+            "       {\n"
+            "         \"account\" : \"accountname\",      (string) DEPRECATED. The account name involved in the transaction, can be \"\" for the default account.\n"
+            "         \"address\" : \"address\",          (string) The dogecoin address involved in the transaction\n"
+            "         \"category\" : \"send|receive\",    (string) The category, either 'send' or 'receive'\n"
+            "         \"amount\" : x.xxx,                 (numeric) The amount in " + CURRENCY_UNIT + "\n"
+            "         \"label\" : \"label\",              (string) A comment for the address/transaction, if any\n"
+            "         \"vout\" : n,                       (numeric) the vout value\n"
+            "         \"fee\": x.xxx,                     (numeric) The amount of the fee in " + CURRENCY_UNIT + ". This is negative and only available for the \n"
+            "                                              'send' category of transactions.\n"
+            "         \"abandoned\": xxx                  (bool) 'true' if the transaction has been abandoned (inputs are respendable). Only available for the \n"
+            "                                              'send' category of transactions.\n"
+            "       }\n"
+            "       ,...\n"
+            "     ],\n"
+            "     \"hex\" : \"data\"         (string) Raw data for transaction\n"
+            "   }\n"
+            "]\n"
+
+            "\nExamples:\n"
+            "\nList all transactions hashes in the wallet that do not have a mempool entry.\n"
+            + HelpExampleCli("liststucktransactions", "") +
+            "\nList all transactions hexs in the wallet that do not have a mempool entry \n"
+            + HelpExampleCli("liststucktransactions", "1") +
+            "\nAs a json rpc call\n"
+            + HelpExampleRpc("liststucktransactions", "")
+        );
+
+    LOCK2(cs_main, pwalletMain->cs_wallet);
+
+    int verbosity = 0;
+    if (request.params.size() > 0)
+        verbosity = request.params[0].get_int();
+
+    isminefilter filter = ISMINE_SPENDABLE;
+    if(request.params.size() > 1)
+        if(request.params[1].get_bool())
+            filter = filter | ISMINE_WATCH_ONLY;
+
+    if(verbosity < 0 || verbosity > 2)
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid verbosity");
+
+    UniValue ret(UniValue::VARR);
+
+    const CWallet::TxItems & txOrdered = pwalletMain->wtxOrdered;
```

Comment:
Minor spacing nit; I'd rather this be `const CWallet::TxItems& txOrdered`. That's not a blocker for this PR (if that's the only thing remaining, I'm okay letting it go).